instance/models/openedx_instance.py

```diff
@@ -117,12 +117,21 @@ def get_load_balancer_configuration(self, triggered_by_instance=False):
         for appserver in active_appservers:
             server_name = "appserver-{}".format(appserver.pk)
             ip_address = appserver.server.public_ip
+            if not ip_address:
+                appserver.logger.error(
+                    "Active appserver does not have a public IP address. This should not happen."
+                    "Calling update_status()"
```

Comment:
We should probably not mention the function name explicitly, in case it has to change; then it'll have to change here, too, and additionally it's not as descriptive.

Better to just describe the process, i.e. "Updating internal status to double check.", or something along those lines.